### PR TITLE
Check GraalVM vendor

### LINKED ISSUE #1102: Inet6Address::cached_scoped_id removed in JDK 11.0.14
See https://bugs.openjdk.java.net/browse/JDK-8216417

The jniconfig-java11.json file includes the method `Inet6Address::cached_scoped_id`, but that can only be applied only until 11.0.13. Based on GraalVM's java version, Substrate will need to remove it, if JDK >= 11.0.14.

### LINKED ISSUE #1104: Detect and warn if GraalVM build is not the required version
Gluon builds of GraalVM are required to use Substrate and GluonFX plugins, however, official builds (release/dev) could be used without any guarantee.
Substrate should check the GraalVM version and warn the user about this.

### PR DESCRIPTION
**Pull request by jperedadnr**
### Issue

Fixes #1104
Fixes #1102

### Progress

- [ ] Change must not contain extraneous whitespace
- [ ] License header year is updated, if required
- [ ] Verify the contributor has signed [Gluon Individual Contributor License Agreement (CLA)](https://docs.google.com/forms/d/16aoFTmzs8lZTfiyrEm8YgMqMYaGQl0J8wA0VJE2LCCY)

### CODE REVIEW

File: src/main/java/com/gluonhq/substrate/Constants.java [hunk 1]
```diff
@@ -117,7 +117,7 @@
     /**
      * Supported GraalVM versions
      */
-    public static final String GRAALVM_MIN_VERSION = "21.0";
+    public static final String GRAALVM_MIN_VERSION = "22.0";
```
Comment (johanvos) [inline src/main/java/com/gluonhq/substrate/Constants.java:120]: shouldn't this be 22.0.0.3 since the cached_scope_id field is removed?

Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/Constants.java:120]: Hmm, we only check major.minor.patch versions, but it could be added

Comment (johanvos) [reply src/main/java/com/gluonhq/substrate/Constants.java:120]: not really needed though with the check on the vendor which is part of this PR

**Review by johanvos — APPROVED** [review]

**Review by tiainen — APPROVED** [review]

File: src/main/java/com/gluonhq/substrate/Constants.java (hunk 1 above)
Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/Constants.java:120]: The check will log a warning only, so those using 22.0.0.2 or lower will need to add the method to the jni list or use runAgent

**Pull request merged**